libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -38,7 +38,7 @@ void
 GCS_MAVLINK::init(AP_HAL::UARTDriver *port, mavlink_channel_t mav_chan)
 {
     // sanity check chan
-    if (mav_chan >= MAVLINK_COMM_NUM_BUFFERS) {
+    if(!valid_chan(mav_chan)) {
```

Comment:
according to the coding style, in control statements like "if", "for", etc there's a space before opening parenthesis, so I changed to `if (!valid_chann)`